ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/pointCloud/RDXROS2ColoredPointCloudVisualizer.java

```diff
@@ -70,32 +67,30 @@ public class RDXROS2ColoredPointCloudVisualizer extends RDXROS2MultiTopicVisuali
 
 
    public RDXROS2ColoredPointCloudVisualizer(String title,
-                                             PubSubImplementation pubSubImplementation,
+                                             ROS2Node ros2Node,
```

Comment:
I would recommend accepting ROS2NodeInterface instead.

Reply:
I don't believe we do that anywhere else in the visualizers. Why should we do that here?

Reply:
I personally don't like the way the inheritance works and would like to avoid using ROS2NodeInterface and ROS2NodeBasics at all

Reply:
In general throughout the codebase it's the practice to use the least interface needed. I think unifying ROS2Node and RealtimeROS2Node would be good but that's another thing

Reply:
In this case, it's never gonna mater though